api/app/tests/common.py

```diff
@@ -130,6 +130,28 @@ async def close(self, *args, **kwargs):
         """Close all http connections."""
 
 
+class MockCFFDRS():
+    """ CFFDRS library mock."""
+    # pylint: disable=invalid-name, no-method-argument, no-self-use
+    def _ROScalc(**kwargs):
+        return 1
+    # pylint: disable=invalid-name, no-method-argument, no-self-use
+
+    def _FMCcalc(**kwargs):
+        return 1
+    # pylint: disable=invalid-name, no-method-argument, no-self-use
+
+    def _SFCcalc(**kwargs):
+        return 1
+
+# pylint: disable=unused-argument
```

Comment:
could you please re-format these lines so that it's clear what the pylint disables apply to? e.g., I'd expect the pylint: disable=invalid-name to be on the line immediately before the function definition, not at the end of the previous function definition. Also, I can't tell what pylint: disable=unused-argument applies to

Reply:
I'd love to but my autosave formatting does this!

Reply:
I'll try to turn off plugins to see which one is causing it.

Reply:
I opted for putting the pylint exception rules on the same line to prevent this issue. We do this in some other places in the codebase.